pyanaconda/installclasses/fedora-server.py

```diff
@@ -0,0 +1,64 @@
+#
+# fedora.py
+#
+# Copyright (C) 2017  Red Hat, Inc.  All rights reserved.
```

Comment:
This is actually Copyright (C) Stephen Gallagher at the moment (I don't assign copyright to Red Hat by default).

If there are reasons that this needs to be assigned, please request it. In general, I prefer to retain my right to have a say in relicensing decisions.